devops/jobs/DockerCI.groovy

```diff
@@ -0,0 +1,99 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    APPS_TO_REPO: a dictionary containing mappings from an edX IDA to the name of its respective GitHub repository
+    APPS_TO_BRANCH: a dictionary containing mappings from an edX IDA to the name of its main GitHub branch
+    CONFIGURATION_REPO_URL: URL of the configuration GitHub repository
+    CONFIGURATION_BRANCH: the main GitHub branch of the configuration repository
+    EDX_REPO_ROOT: the URL of the edX GitHub organization
+    CONFIG_JSON_FILE_CREDENTIAL_ID: ID of the credentials set up in Jenkins to log into DockerHub
```

Comment:
I spoke to Feanil, and he said that the rationale was that, if other teams/communities want to use this job, they may want/need to use other credentials or not have credentials at all, so it would be valuable for it to be overridable.